search/ranker.cpp

```diff
@@ -499,24 +507,44 @@ void Ranker::GetBestMatchName(FeatureType const & f, string & name) const
   UNUSED_VALUE(f.ForEachName(bestNameFinder));
 }
 
-void Ranker::ProcessSuggestions(vector<IndexedValue> & vec) const
+void Ranker::MatchForSuggestions(strings::UniString const & token, int8_t locale,
+                                 string const & prologue)
+{
+  for (auto const & suggest : m_suggests)
+  {
+    strings::UniString const & s = suggest.m_name;
+    if (suggest.m_prefixLength <= token.size()
+        && token != s                  // do not push suggestion if it already equals to token
+        && suggest.m_locale == locale  // push suggestions only for needed language
+        && strings::StartsWith(s.begin(), s.end(), token.begin(), token.end()))
```

Comment:
s is a UniString, token is a UniString, we have StartsWith for UniStrings, please, use it.

Reply:
done